src/Http/Http.Abstractions/src/HttpMethods.cs

```diff
@@ -5,12 +5,15 @@
 
 namespace Microsoft.AspNetCore.Http
 {
+    /// <summary>
+    /// Contains methods to verify the request method of an HTTP request. 
+    /// <summary/>
```

Comment:
Ah, actually, this isn't well-formed XML so it doesn't build. It should be `</summary>`. The same goes for all the other XML tags. Can you do a pass to make sure the XML is all valid?

Reply:
oh that was dumb on my part. I will get those fixed.

Reply:
No problem, I reviewed all the code and completely missed it too ;). Good thing we have the build!